pootle/i18n/formatter.py

```diff
@@ -15,12 +15,14 @@
 from django.utils.translation import get_language, to_locale
 
 
-def get_locale_formats():
-    for language in [get_language(), settings.LANGUAGE_CODE, 'en-us']:
+def get_locale_formats(locale=None):
+    languages = [locale] if locale else []
+    languages += [get_language(), settings.LANGUAGE_CODE, 'en-us', 'en']
```

Comment:
Was there a specific reason to fallback to `en`?  Since our app is in `en-us` it doesn't make much sense to me.  And I suspect in CLDR `en-us == en` so kinda pointless step IIUC and just confuses the matter with no justification in the code or the commit